db/re/item_db_etc.yml

```diff
@@ -251,7 +251,7 @@ Body:
     AegisName: Blue_Gemstone
     Name: Blue Gemstone
     Type: Etc
-    Buy: 1500
+    Buy: 450
```

Comment:
Are you sure ?

I check in http://kafra.kr/#!/en/KRO/itemdetail/717 buy price shop still 1500

Reply:
Checked in kRO
![image](https://github.com/user-attachments/assets/4115ac9f-60af-485f-a527-99b2cfb1f853)
![image](https://github.com/user-attachments/assets/3c9af018-f882-4bed-9623-b2e4c6d599fb)